dev/com.ibm.ws.security.jwt/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -106,4 +106,19 @@ elapsedNBF.desc=Identifies the elapsed time until JWT will start to be accepted
 
 #Do not translate "amr"
 amrValues=Authentication methods references in string array
-amrValues.desc= Specifies Authentication Methods References so it can enforce step-up authentication and know how the user is authenticated. Specify a comma separated list of valid amr claim values to be used.
\ No newline at end of file
+amrValues.desc= Specifies Authentication Methods References so it can enforce step-up authentication and know how the user is authenticated. Specify a comma separated list of valid amr claim values to be used.
```

Comment:
`Specifies Authentication Methods References so it can enforce step-up authentication`

what does "it" refer to in this context?

Reply:
I'm actually not entirely sure - that property was written and delivered elsewhere. Best guess from the context is that "it" would be our JWT consumer functionality.

Reply:
- capitalization of "Authentication Methods References" should be consistent in the name and description- 
`Authentication Methods References in string array`

suggested edit
`Specifies Authentication Methods References (amr) to enforce step-up authentication and indicate how the user is authenticated. Specify a comma separated list of valid amr claim values.`

Reply:
There's a bit of a catch-22 in our metatype properties validation when it comes to proper names and acronyms. The name is required to be in sentence case (unless a specific exception is made for individual words) while the description is a bit more relaxed. If I wanted to put "International Business Machines" in the name, for instance, by default it would have to be "International business machines" to comply with the validation rules even though that's not actually proper. In any case, I don't think anything is lost with lowercasing the phrase in the description so I've gone ahead and done that.